src/extension.ts

```diff
@@ -11,6 +11,17 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+'use strict';
+
+import path = require('path');
+import { visit } from 'jsonc-parser';
+
+module.exports = {
+	activate
+};
+
+let pendingFooJsonDecoration: string | number | NodeJS.Timeout | undefined;
```

Comment:
Does this need to include `string | number`?

Reply:
Probably not, no idea where that came from